docs/maintainers/faq.md

```diff
@@ -0,0 +1,150 @@
+
+Not all Concept Exercises have to designed from scratch, it is perfectly reasonable for tracks to adapt Concept Exercises that were created for other tracks. The important part here is that when adapting existing Concept Exercises, they should be modified to best fit the track that is adapting them.
+
+For some basic Concept Exercises, it might make sense to design them to teach two Concepts. The additional Concept must be one that is very simple and can be taught well in the context of the primary Concept. As an example, a basic "Integers" exercise could decide to also teach the basic "Conditionals" Concept. Note though that have more than one Concept should be the exception, not the rule.
+
+## What files make up a Concept Exercise?
+
+A Concept Exercise has two types of files. The first are the general, language-agnostic files, which are required for all tracks. You can find a description of these files in the generic [how to implement a concept exercise][how-to-implement-a-concept-exercise] section. The second type of files are language-specific, and as such are different for each track. A description of these language-specific files will be added to each track. Here are the [C# track's language-specific instructions][how-to-implement-a-concept-exercise-csharp].
+
+## How to implement a Concept Exercises?
+
+Implementing a Concept Exercise should in principle be a case of following the detailed instructions in the Concept Exercise's corresponding GitHub issue.
+
+If your track has already implemented one or more Concept Exercises, we highly recommend you taking a look at them too, to see how they were implemented.
+
+## How many Concept Exercises should be created?
+
+We're aiming for about 50 Concept Exercises, although that does depend on the language and how many concepts it has (some languages are more complex than others). There is no fixed number here, it all depends on the language and how many Concepts it has. For the more complex languages with lots of concepts, the number could easily go to 100 Concept Exercises.
+
+As each Concept Exercise will have detailed instructions in the form of a GitHub issue, we're hoping to crowd-source the actual implementation of the Concept Exercises. Designing the Concept Exercises and creating the corresponding GitHub issues is a responsibility of the maintainers though.
+
+## Is there a 1-1 mapping from Concept to Concept Exercise?
+
+In general: yes. However, for very complex Concepts, it can make sense to have multiple smaller Concept Exercises instead of one huge Concept Exercise.
+
+We highly recommend reading [this thread][concept-to-exercise-mapping] for a nice discussion on this topic.
+
+## How will mentoring work for Concept Exercises?
+
+There will be no mentoring on Concept Exercises in v3. The process will be fully automated, and will work as follows:
```

Comment:
Should there be some placeholder text here, to show that "as follows:" doesn't have the necessary information yet, but we expect it to be written?  Perhaps "TO BE WRITTEN" or something so that it brings attention to itself.

Reply:
Or is it only appearing to be that way because the heading level of the next section(s) is the same as this section?

Reply:
Good catch. The second heading should be removed. Will update.